draft-ietf-rtcweb-jsep.xml

```diff
@@ -3222,6 +3181,49 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           below.</t>
         </list></t>
       </section>
+      <section title="Processing a Rollback" anchor="sec.processing-a-rollback">
+        <t>A rollback may be performed if the PeerConnection is in any
+        state except for "stable". This means that both offers and provisional
+        answers can be rolled back. If a rollback is attempted in the "stable"
+        state, processing MUST stop and an error MUST be returned.The effect
+        MUST be the same regardless of whether setLocalDescription or
+        setRemoteDescription is called.</t>
+        
+        <t>In order to process rollback, a JSEP implementation
+        abandons the current offer/answer transaction, sets the signaling
+        state to "stable", and sets the pending local and/or remote
+        description (see <xref target="sec.pendinglocaldescription" /> and
+        <xref target="sec.pendingremotedescription" />) to null.
+        Any resources or candidates that were allocated by the
+        abandoned local description are discarded; any media that
+        is received will be processed according to the previous
+        local and remote descriptions. Rollback can only be used to
```

Comment:
This discussion of states should probably move to the para above that discusses valid states.